README.md

```diff
@@ -40,13 +40,29 @@ Money.new(1000, "USD") != Money.new(1000, "EUR")   #=> true
 # Arithmetic
 Money.new(1000, "USD") + Money.new(500, "USD") == Money.new(1500, "USD")
 Money.new(1000, "USD") - Money.new(200, "USD") == Money.new(800, "USD")
-Money.new(1000, "USD") / 5                     == Money.new(200, "USD")
 Money.new(1000, "USD") * 5                     == Money.new(5000, "USD")
 
+# Splitting money evenly
+Money.new(1000, "USD").split(2)            == [Money.new(500, "USD"), Money.new(500, "USD")]
+Money.new(1000, "USD").split(3)            == [Money.new(333.34, "USD"), Money.new(333.33, "USD"), Money.new(333.33, "USD")]
+Money.new(1000, "USD").calculate_splits(2) == { Money.new(500, "USD") => 2 }
+Money.new(1000, "USD").calculate_splits(3) == { Money.new(333.34, "USD") => 1, Money.new(333.33, "USD") =>2 }
+
+# Allocating money proportionally
+Money.new(1000, "USD").allocate([0.50, 0.25, 0.25])               == [Money.new(500, "USD"), Money.new(250, "USD"), Money.new(250, "USD")]
+Money.new(1000, "USD").allocate([Rational(2, 3), Rational(1, 3)]) == [Money.new(666.67, "USD"), Money.new(333.33, "USD")]
+
+## Allocating up to a cutoff
+Money.new(1000, "USD").allocate_max_amounts([50, 30, 30])    == [Money.new(50, "USD"), Money.new(30, "USD"), Money.new(30, "USD")]
```

Comment:
To avoid confusion as to what this method is doing, this could be a better happy case example, as the total sums up to 1000
```suggestion
Money.new(1000, "USD").allocate_max_amounts([500, 300, 200])    == [Money.new(500, "USD"), Money.new(300, "USD"), Money.new(200, "USD")]
```